force table to fit in the article width (#717)
pydata/pydata-sphinx-theme: docs/demo/kitchen-sink/lists-and-tables.rst

| body row 1 | column 2   | column 3  | body row 1 | column 2   | column 3  | body row 1 | column 2   | column 3  | body row 1 | column 2   | column 3  |
+------------+------------+-----------+------------+------------+-----------+------------+------------+-----------+------------+------------+-----------+

Giant Content
^^^^^^^^^^^^^

> If the table is filled with giant content, the template will wrap you words to fit the page width.

Reviewer: ```suggestion
If table cells are filled with a lot of words, they should wrap in order to fit the page width.
```
